### PR TITLE
Add Vibrate Button

### PR DESCRIPTION
**Pull request by adrsh79086**
Issue:

### CODE REVIEW

**Review by dsnehasish74 — CHANGES_REQUESTED** [review]
Thank you for your work but can you please add this button in alphabetical order

**Comment by adrsh79086** [comment]
> Thank you for your work but can you please add this button in alphabetical order

okey i will do

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -224,6 +224,11 @@ $(document).ready(function () {
           label: "Liquid",
           classes: "liquid-btn",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:227]: As @dsnehasish74 said please place it in the correct alphabetical order. Also, capitalize the first letter in the label

**Review by shahednasser — CHANGES_REQUESTED** [review]
Please change the animation to happen on hover instead.

**Comment by glebkema** [comment]
One more thanks for your PR! 

Please start your include less file with an underscore: `_ vibrate.less`. This helps Gulp distinguish the include files from the main file.

**Comment by shahednasser** [comment]
Also as @glebkema pointed out, please rename the `vibrate.less` to `_vibrate.less` and change the import as well.

File: src/components/_animated.less [hunk 2]
```diff
@@ -25,3 +25,4 @@
 @import "animated/_loading-btn";
 @import "animated/_spring.less";
 @import "animated/_papercurl.less";
+@import "animated/vibrate.less";
```
Comment (glebkema) [inline src/components/_animated.less:28]: One more thanks for your PR!

Please start your include less file with an underscore: _vibrate.less. It helps Gulp distinguish the include files from the main file.

File: assets/js/buttons-examples.js [hunk 3]
```diff
@@ -345,6 +345,11 @@ $(document).ready(function () {
           label: "Transparent On Hover",
           classes: "transparent-btn",
         },
+        {
+          type: "Vibrate",
```
Comment (dsnehasish74) [inline assets/js/buttons-examples.js:349]: please make the first letter of the type small .

**Review by dsnehasish74 — COMMENTED** [review]
And please make the file name _vibrate.less instead of _Vibrate.less

**Pull request closed without merging**